### PR TITLE
I35 explore data app

### LINKED ISSUE #35: Create Explore data app
Following the Django principle "an app should only do one thing" this app handles the first "Explore data" view for the NCA (non-compartmental PK analysis) and the AUCE (area under the curve estimation) workflow. 

The NCA and AUCE will be handled in separate apps. This makes the PKPDApp more modular.

### PR DESCRIPTION
**Pull request by DavAug**
Closes #35 .

This is just the explore data view leading to the NCA and AUCE analysis workflows

### CODE REVIEW

File: pkpdapp/explore_data/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        response = self.client.get(reverse('explore_data:index'))
+        contents = 'Analyse Pharmacokinetics'
+        self.assertContains(response, contents)
+        contents = 'Analyse Pharmacodynamics'
+        self.assertContains(response, contents)
+
+    def test_index_does_not_contain_incorrect_html(self):
+        response = self.client.get(reverse('explore_data:index'))
+        self.assertNotContains(
+            response, 'Hi there! I should not be on the page.')
```
Comment (martinjrobins) [inline pkpdapp/explore_data/tests/test_views.py:34]: Is this test necessary? I can't see how it would ever fail?

**Review by martinjrobins — COMMENTED** [review]
Looks good @DavAug, one query below

File: pkpdapp/explore_data/tests/test_views.py (hunk 1 above)
Comment (DavAug) [reply pkpdapp/explore_data/tests/test_views.py:34]: Agreed :D, I don't really know why I have that in there either. I'll remove it.

**Comment by DavAug** [comment]
@martinjrobins 
Thanks for the review Martin. I removed the test :)

**Review by martinjrobins — APPROVED** [review]
excellent :)

**Pull request merged**